app/models/like.rb

```diff
@@ -1,4 +1,11 @@
 class Like < ApplicationRecord
-  belongs_to :author
+  after_save :update_likes_counter
+  belongs_to :author, class_name: 'User'
```

Comment:
👏 Good job so far! I appreciate your efforts in addressing the optional changes suggested by the previous reviewer, which is beneficial for project enhancement. However, I recommend updating your schema and model structure slightly. While it's true that in this project, you are using one user table, it serves multiple roles—for posts as the author and for likes and comments as the user. This approach is meaningful.

- [ ] [Optional] In your project, you consistently use the term "author" everywhere. To align with the ER diagram and enhance clarity, I suggest updating your schema and models to use the term "user" in places where it represents the entity involved in likes and comments.
![image](https://github.com/ReemMohamedAbdelfatah/Blog_APP/assets/55840999/1c78c07e-c316-42c4-96c5-84b5a8ec443d)

By making these adjustments, you'll ensure consistency with the ER diagram and improve the overall understanding of your project structure. Keep up the good work! 🚀👍